pydis_core/utils/__init__.py

```diff
@@ -45,10 +47,12 @@ def apply_monkey_patches() -> None:
     interactions,
     logging,
     members,
+    messages,
     paste_service,
     regex,
     scheduling,
     unqualify,
+    pagination
```

Comment:
```suggestion
    messages,
    paste_service,
    pagination,
    regex,
    scheduling,
    unqualify,
```
to keep it alphabetised